src/notifications/batterynotifier.cpp

```diff
@@ -153,6 +158,12 @@ void BatteryNotifier::prepareNotification()
             checkChargingTimer->start();
         }
     }
+    if (isStateChanged) {
+        if (!isAnyChargingState)
```

Comment:
@chriadam I am using kernel conventions (no braces for 1-liners) to avoid extra syntax noise. But not a big deal to change it here, maybe it will be more consistent with the rest of the lipstick code.